virtual_rainforest/data_variables.toml

```diff
@@ -0,0 +1,385 @@
+[[variable]]
+name = "air_temperature"
+description = "air temperature profile"
+unit = "C"
+source = "abiotic"
+used_by = [ "abiotic", "plants", "animals"]
+
+[[variable]]
+name = "air_temperature_ref"
+description = "air temperature at reference height (2m)"
+unit = "C"
+source = "external"
+used_by = [ "abiotic"]
+
+[[variable]]
+name = "albedo_shortwave"
+description = "shortwave light albedo"
+unit = "-"
+source = "external"
+used_by = [ "abiotic"]
+
+[[variable]]
+name = "albedo_vis"
+description = "visible light albedo"
+unit = "-"
+source = "external"
+used_by = [ "abiotic"]
+
+[[variable]]
+name = "animal_respiration"
+description = "animal respiration aggregated over all functional types"
+unit = "ppm"
+source = "animal"
+used_by = [ "abiotic", "animals"]
+
+[[variable]]
+name = "atmospheric_co2_ref"
+description = "CO2 concentration at reference height (above canopy)"
+unit = "ppm"
+source = "external"
+used_by = [ "abiotic"]
+
+[[variable]]
+name = "atmospheric_pressure"
+description = "atmospheric pressure profile"
+unit = "kPa"
+source = ["abiotic"] 
+used_by = [ "abiotic", "plants"]
+
+[[variable]]
+name = "atmospheric_pressure_ref"
+description = "atmospheric pressure at reference height (2m)"
+unit = "kPa"
+source = ["external"] 
+used_by = [ "abiotic"]
+
+[[variable]]
+name = "bulk_aerodynamic_resistance"
+description = "bulk aerodynamic resistance"
+unit = "s m-1"
```

Comment:
We might have to think about how we express this type of unit, I think calling this "m^-1" might be better. Might be somewhat tricky to address now though as it very much depends on a what can be read correctly (which I'm not totally sure on)

Reply:
Or if it even needs to be parsed. I'm not sure how we'd actually use parsed units unless we attach units to everything.

Reply:
Good point, I think it is still useful at the moment while we use it as a lookup table, too